ezcord/bot.py

```diff
@@ -288,6 +294,26 @@ def _manage_cogs(
         self._cog_count_log(custom_log_level, log, loaded_cogs, log_color)
         return cogs
 
+    def load_extension(self, name: str, **kwargs):
+        """Loads an extension with configurable error handling.
+
+        This method attempts to load a bot extension. The behavior on error depends
+        on the `safe_loading` setting in the ``ezcord.Bot`` instance.
+
+        Parameters
+        ----------
+        name:
+            The name of the extension to load.
+        **kwargs:
+            Additional parameters to pass to load_extension.
+        """
+        try:
+            super().load_extension(name, **kwargs)
+        except Exception as e:
+            if not self.safe_loading:
+                raise
+            self.logger.error(f"Failed to load extension '{name}' ({e})")
```

Comment:
I'm not sure if we should display the traceback here, what do you think?
```suggestion
            self.logger.error(f"Failed to load extension '{name}'", exc_info=e.__cause__)
```

Reply:
Sounds good 👍